server/service/integration_mdm_setup_experience_test.go

```diff
@@ -0,0 +1,1614 @@
+package service
```

Comment:
Double-checking, are there any new tests added in this file or any substantive changes to the tests that were copied over?